src/shared/components/post/post-listing.tsx

```diff
@@ -1780,6 +1755,10 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
     setupTippy();
   }
 
+  // handleShowPreview(i: PostListing) {
```

Comment:
Any reason to keep this commented out function?

Reply:
Ah, no -- was planning ahead. Deleted.